scripts/scr_company_view/scr_company_view.gml

```diff
@@ -52,18 +52,21 @@ function scr_company_view(company) {
 	sel_uni[1]="Command";
 
 	// This sets up the mans, but not the vehicles
-	var company_length = array_length(obj_ini.name[company]);
 	// Company_lenght is 501, so we check with 500
-	for (var v = 1; v < (company_length-1); v++){
+	for (var v = 0; v <= 500; v++){
```

Comment:
can we keep the old version, i.e. check company length? we really don't want to hardcode stuff anymore